scylladb/sphinx-scylladb-theme
# Add collapsible sidebar

Closes https://github.com/scylladb/sphinx-scylladb-theme/issues/193

I moved the selector to the bottom and inside the sidebar to not have conflicts with the scrollbar or the multiversion selector.

## How to test this PR

1. Clone this PR. For more information, see [Cloning pull requests locally](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/checking-out-pull-requests-locally).

2. Enter the docs folder, and run:

```
make preview
````

3. Open http://127.0.0.1:5500/ with your favorite browser, and enter an internal page.


The sidebar collapsed should look like:

![image](https://user-images.githubusercontent.com/9107969/145175629-cccbb4e0-d850-480e-91c6-e7b05df20052.png)

The sidebar expanded should look like:

![image](https://user-images.githubusercontent.com/9107969/145175207-ac5eba7f-88d3-49dd-83cd-e59dea89140d.png)
